io/src/main/scala/com/velocidi/apso/io/LocalFileDescriptor.scala

```diff
@@ -22,6 +22,8 @@ case class LocalFileDescriptor(initialPath: String) extends FileDescriptor with
 
   lazy val name: String = file.getName
 
+  def isLocal: Boolean = true
```

Comment:
Nitpick:
```suggestion
  val isLocal: Boolean = true
```

Reply:
Is the constant extra space for every file descriptor worth it?

Reply:
Well... I think we're talking about an extra constant byte (more or less), and I am assuming there is a (very small) performance improvement by it being a `val`, but that assumption might be thrown away by some optimization on the vm... so, I am perfectly ok on not having this as a `def`.

I tend to prefer a `val` because it more accuretly depicts that this should be constant for the file descriptor in question.

Reply:
I suppose it's OK to use `val` for this.